libs/output-mapping/src/Storage/StoragePreparer.php

```diff
@@ -34,7 +36,7 @@ public function prepareStorageBucketAndTable(
         $destinationTableInfo = $this->getDestinationTableInfoIfExists(
             $processedSource->getDestination()->getTableId(),
         );
-        if ($destinationTableInfo !== null) {
+        if (!$hasNewNativeTypeFeature && $destinationTableInfo !== null) {
```

Comment:
👍

Reply:
👎  V tom IFu se dělá modifikace struktury + delete rows.
Neměla by se jen vynechat ta modifikace struktury?

Jak se to postupně upravuje všechno do podmínek má/nemá NNTF, tak si skoro říkým, že by ten delete rows měl být někde jinde 🤔

Reply:
ok